application/console/components/AwsStartupAction.php

```diff
@@ -62,6 +73,26 @@ public function performAction()
             $logger->appbuilderExceptionLog($logException, $e);
         }
     }
+    private function copyDefaultProjectFolder($logger)
+    {
+        try {
+            $prefix = Utils::getPrefix();
+            echo "[$prefix] AwsStartupAction: copyDefaultProjectFolder" . PHP_EOL;
+            $s3 = new S3();
+            $currentDir = getcwd();
+            $sourceFolder = '/data/console/views/cron/scripts/project_default';
```

Comment:
I see you got `getcwd` but used a full path that would only work in docker.  Did you intent to build a path?

Reply:
I copied that from the other pr, saw it was not used and still forgot to take it out.  I will correct that